src/main/java/org/generationcp/middleware/service/api/releasenote/ReleaseNoteService.java

```diff
@@ -10,6 +10,6 @@
 
 	Optional<ReleaseNote> getLatestReleaseNote();
 
-	void dontShowAgain(Integer userId);
+	void showAgain(Integer userId, boolean showAgain);
```

Comment:
@cuenyad I think this change is not required for this feature, right? maybe wrong branch?